src/query/storages/fuse/fuse/src/operations/truncate.rs

```diff
@@ -72,11 +72,15 @@ impl FuseTable {
             let db_name = ctx.get_current_database();
 
             catalog
-                .update_table_meta(&tenant, &db_name, UpdateTableMetaReq {
-                    table_id,
-                    seq: MatchSeq::Exact(table_version),
-                    new_table_meta,
-                })
+                .update_table_meta(
+                    &tenant,
+                    self.table_info.db_type.clone(),
+                    UpdateTableMetaReq {
+                        table_id,
+                        seq: MatchSeq::Exact(table_version),
+                        new_table_meta,
+                    },
+                )
```

Comment:
`TableInfo` contains both tenant and `db_type`, it looks like `update_table_meta` should accept a `TableInfo` argument. in next PR 🤔